conanfile.py

```diff
@@ -37,7 +41,8 @@ def build(self):
             cmake.definitions["FMT_TEST"] = False
             cmake.definitions["FMT_INSTALL"] = True
             cmake.definitions["BUILD_SHARED_LIBS"] = self.options.shared
-            cmake.definitions["FMT_LIB_DIR"] = "lib"
+            if self.options.fPIC:
```

Comment:
fPIC should not be listed on Windows build, so you could replace the line 44 by: